lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +253,30 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = article_bs['href']
+        if isinstance(url, str):
+            return url
+        return url[0]
```

Comment:
can you explain why you return either url directly or the first element in url?

Reply:
Because it can find some elements, but we need only one, so it is string if it finds one element and it is a list if it finds several.